larpandoracontent/LArHelpers/LArVertexHelper.cc

```diff
@@ -58,4 +59,21 @@ LArVertexHelper::ClusterDirection LArVertexHelper::GetClusterDirectionInZ(
     throw StatusCodeException(STATUS_CODE_FAILURE);
 }
 
+//-----------------------------------------------------------------------------------------------------------------------------------------
+
+bool LArVertexHelper::IsInFiducialVolume(const CartesianVector &vertex, const std::string &detector)
+{
+    if (detector == "dune_fd_hd")
```

Comment:
`==` uses `compare(string)` under the hood so you could save 0.000001 CPU cycles by using that.